pkg/api/v1beta3/types.go

```diff
@@ -521,7 +521,9 @@ type Container struct {
 	// Required.
 	Image string `json:"image" description:"Docker image name"`
 	// Optional: Defaults to whatever is defined in the image.
-	Command []string `json:"command,omitempty" description:"command argv array; not executed within a shell; defaults to entrypoint or command in the image; cannot be updated"`
+	Command []string `json:"command,omitempty" description:"entrypoint array; not executed within a shell; defaults to the image's entrypoint; cannot be updated"`
+	// Optional: Defaults to the image's entrypoint
+	Args []string `json:"args,omitempty" description:"command array; defaults to image's cmd; arguments to the entrypoint"`
```

Comment:
Somewhere we should say a bit more about how Command and Args are combined.

Otherwise, the API changes LGTM.